package.json

```diff
@@ -198,6 +198,7 @@
         "axe-core": "4.10.0",
         "babel-jest": "^29.0.0",
         "blob-polyfill": "^9.0.0",
+        "core-js": "^3.38.1",
```

Comment:
This is to add a polyfill to avoid the test crashing, ie. https://github.com/element-hq/matrix-react-sdk/pull/44/files#diff-0bd42c32df0a1d0f3ebbfb8eebc1c08958e613a9e85ff581f99ccd7f00b4ea34R11